test-data/run.test

```diff
@@ -3197,3 +3197,42 @@ hello!
 contextmanager: exited
 contextmanager: entering
 contextmanager: exited
+
+[case testAnyAll]
+from typing import Iterable
+
+def call_any(l: Iterable[int], val: int = 0) -> bool:
+    return any(i == val for i in l)
```

Comment:
Add a test that tests using the result of an `any` or `all` within a function without immediately returning it.